lib/config.coffee

```diff
@@ -90,6 +90,7 @@ exports.generate = (options, params = {}) ->
 		mixpanelToken: options.mixpanel.token
 		apiKey: options.apiKey
 
+		connectivity: params.connectivity or 'connman'
```

Comment:
I believe `or 'connman'` is not needed, since you're already declaring this as a default value in `schema.coffee`.